app/services/hmpps_api/prison_api/agencies_api.rb

```diff
@@ -0,0 +1,21 @@
+module HmppsApi
+  module PrisonApi
+    class AgenciesApi
+      extend PrisonApiClient
+      HOSPITAL_AGENCY_TYPE = 'HSHOSP'.freeze
+      def self.agencies_by_type(type)
+        route = "/agencies/type/#{type}"
+        data = client.get(route, cache: true)
+        data.map { |agency|
+          { agency_type: agency['agencyId'], description: agency['description'], active: (!!agency['active']) }
```

Comment:
It's generally unnecessary to do `!!` in Ruby - what type would `agency['active']` be?

Reply:
This was actually recommended by the IDE, which is looking at the Rubocop params. 
The api is here https://api-dev.prison.service.justice.gov.uk/swagger-ui/#/agencies/getAgencies and the value is a boolean, so it should be fine to change

Reply:
Hmm, okay, well it certainly looks odd to do `!!` here, and I'm surprised it was recommended by the IDE!